ui_testing/elements.py

```diff
@@ -191,7 +191,17 @@
                                    'value': 'a',
                                    'order': 1},
         'new_test_plan': {'method': 'link_text',
-                          'value': 'New Test Plan'}
+                          'value': 'New Test Plan'},
+        'right_menu': {'method': 'xpath',
+                        'value': '//*[@id="custom-accordion-panel"]/div/a/i'},
```

Comment:
use relative xpath or another methods if possible

Reply:
I'm going to remove it, I no longer need it.